portfolio/src/main/webapp/script.js

```diff
@@ -110,3 +111,12 @@ async function deleteAll() {
   fetch("/delete-data", { method: "POST" });
   window.location.reload();
 }
+
+async function loginStatus(){
+  const response = await fetch("/login");
+  const login= await response.text();
+  console.log(login);
+  if(login.includes("true")){
+    document.getElementById("login").style.display = "block";
```

Comment:
Nit: preference for adding/removing a `.hide` class rather than directly changing the inline style. This gives more flexibility to dynamically hide other elements on the page and makes it easier to tell what is initially hidden by reading the HTML.